libs/designer/src/lib/ui/templates/parameters/displayParameters.tsx

```diff
@@ -0,0 +1,55 @@
+import { List, Text } from '@fluentui/react';
+import type { TemplateParameterUpdateEvent } from '@microsoft/designer-ui';
+import { updateTemplateParameterValue } from '../../../core/state/templates/templateSlice';
+import { useIntl } from 'react-intl';
+import type { AppDispatch, RootState } from '../../../core/state/templates/store';
+import { useDispatch, useSelector } from 'react-redux';
+import { TemplateparameterField } from '@microsoft/designer-ui';
+import type { Template } from '@microsoft/logic-apps-shared';
+
+export const DisplayParameters = () => {
+  const dispatch = useDispatch<AppDispatch>();
+  const { parameters } = useSelector((state: RootState) => state.template);
+  const intl = useIntl();
+  const validationErrors = parameters.validationErrors;
+  const parametersDefinition = Object.entries(parameters.definitions).map(([key, value]) => ({ id: key, ...value }));
+
+  const onUpdateParameterValue = (event: TemplateParameterUpdateEvent) => dispatch(updateTemplateParameterValue(event));
+
+  const titleText = intl.formatMessage({
+    defaultMessage: 'Parameters',
+    id: 'X7X5ew',
+    description: 'Workflow Parameters Title',
+  });
+
+  const renderParameter = (item?: Template.ParameterDefinition): JSX.Element => {
+    if (!item) {
+      return <></>;
+    }
+    return (
+      <div className="msla-workflow-parameter">
```

Comment:
NIT: whenever rendering list of items its good to put a `key` attribute to the top parent tag, sometimes it messes it up with the re-rendering or update of fields.